_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -30,20 +30,20 @@ This rule applies to any `headers` attribute specified on a [`cell`][] within a
 
 ## Expectation 1
 
-Each target attribute is [a set of space separated IDs][], each of which is the ID of an element, that is a [`cell`][] of the same [`table`][].
+Each target's `headers` [attribute value][] is a [set of space separated tokens][]. Each token is the value of the `id` attribute of an element, that is a [`cell`][] of the same [`table`][].
 
 **Note:** `headers` attribute referencing elements that are non-existent or not in the table are ignored when [assigning header cells (step 3, first case, point 2)](https://html.spec.whatwg.org/multipage/tables.html#algorithm-for-assigning-header-cells).
 
 ## Expectation 2
 
-Each target attribute is [a set of space separated IDs][], none of which is the ID of the element on which the test target is specified.
+Each target's `headers` [attribute value][] is a [set of space separated tokens][] that is not the value of `id` attribute of the element on which the test target is specified.
```

Comment:
We need to keep the "none of which" formulation. `<td id="foo" headers="foo bar">`, the set {`foo`, `bar`} is not the value `foo`, so that would pass the expectation.
```suggestion
Each target's [attribute value][] is a [set of space separated tokens][], and none of these tokens is the value of `id` attribute of the element on which the test target is specified.
```